trilby-cli/flake.nix

```diff
@@ -49,30 +41,21 @@
           haskell = prev.haskell // {
             packageOverrides = lib.composeExtensions
               prev.haskell.packageOverrides
-              (hfinal: hprev:
-                with prev.haskell.lib.compose;
-                {
-                  hnix = dontCheck (hfinal.callCabal2nix "hnix" inputs.hnix { });
-                  hnix-store-core = doJailbreak (hfinal.callCabal2nix "hnix-store-core" "${inputs.hnix-store}/hnix-store-core" { });
-                  hnix-store-db = hfinal.callCabal2nix "hnix-store-db" "${inputs.hnix-store}/hnix-store-db" { };
-                  hnix-store-nar = hfinal.callCabal2nix "hnix-store-nar" "${inputs.hnix-store}/hnix-store-nar" { };
-                  hnix-store-readonly = hfinal.callCabal2nix "hnix-store-readonly" "${inputs.hnix-store}/hnix-store-readonly" { };
-                  hnix-store-remote = hfinal.callCabal2nix "hnix-store-remote" "${inputs.hnix-store}/hnix-store-remote" { };
-                  hnix-store-tests = hfinal.callCabal2nix "hnix-store-tests" "${inputs.hnix-store}/hnix-store-tests" { };
-                  "${pname}" = (hfinal.callCabal2nix "trilby-cli" src { }).overrideAttrs
-                    (attrs: {
-                      outputs = (attrs.outputs or [ ]) ++ [ "bin" ];
-                      meta = attrs.meta // {
-                        outputsToInstall = [ "out" "bin" ];
-                      };
-                      postInstall = ''
-                        ${attrs.postInstall or ""}
-                        install -Dt $bin/bin $out/bin/*
-                      '';
-                    });
-                }
-              );
+              (hfinal: hprev: {
+                "${pname}" = (hfinal.callCabal2nix pname src { }).overrideAttrs
+                  (attrs: {
+                    outputs = (attrs.outputs or [ ]) ++ [ "bin" ];
+                    meta = attrs.meta // {
+                      outputsToInstall = [ "out" "bin" ];
+                    };
+                    postInstall = ''
+                      ${attrs.postInstall or ""}
+                      install -Dt $bin/bin $out/bin/*
```

Comment:
I think we don't need this line -- the default builder should do this automatically if "bin" is in `outputs`.

Reply:
I'd love to omit it, but it seems the default builder doesn't do it:

```
error: builder for '/nix/store/sz4pgwwcswvi8m99fbwx4hj5nyykkfi6-trilby-cli-0.1.0.0.drv' failed to produce output path for output 'bin' at '/nix/store/sz4pgwwcswvi8m99fbwx4hj5nyykkfi6-trilby-cli-0.1.0.0.drv.chroot/nix/store/8vy0gf18rlcjb4qfc21p9wsmvydicag6-trilby-cli-0.1.0.0-bin'
```